# service for getting the environment description

---
ros_rl/EnvDescMsg envDesc